server/eclipse-project/src/main/java/au/org/democracydevelopers/corla/model/vote/IRVBallotInterpretation.java

```diff
@@ -1,25 +1,146 @@
-/*
-Democracy Developers IRV extensions to colorado-rla.
+package au.org.democracydevelopers.corla.model.vote;
 
-@copyright 2024 Colorado Department of State
+import org.apache.commons.lang.StringEscapeUtils;
+import us.freeandfair.corla.model.CastVoteRecord;
+import us.freeandfair.corla.model.Contest;
+import us.freeandfair.corla.persistence.PersistentEntity;
+import us.freeandfair.corla.persistence.StringListConverter;
 
-These IRV extensions are designed to connect to a running instance of the raire 
-service (https://github.com/DemocracyDevelopers/raire-service), in order to 
-generate assertions that can be audited using colorado-rla.
+import javax.persistence.*;
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Stream;
 
-The colorado-rla IRV extensions are free software: you can redistribute it and/or modify it under the terms
-of the GNU Affero General Public License as published by the Free Software Foundation, either
-version 3 of the License, or (at your option) any later version.
+/**
+ * A record of the interpretation of invalid IRV ballots. Used for reporting, not for the actual
+ * auditing logic. Every time an invalid IRV vote is encountered, whether in the initial upload or
+ * during the auditing steps, a record is made of the CVR ID, the county, contest name, raw vote,
+ * and our interpretation of that vote as an ordered list of candidate names.
+ * This is output as a report called IRVBallotInterpretationReport.
+ */
```

Comment:
Check whether we need this to implement the Serializable interface in addition to PersistentEntity. If not, some of the methods can be removed if we don't implement it.

Reply:
I got rid of Serizalizable but unfortunately those 3 overridden messages are still needed for PersistentEntity.